Library/Homebrew/test/cmd/uninstall_spec.rb

```diff
@@ -19,6 +19,41 @@
   end
 end
 
+describe "brew uninstall cask", :integration_test, :needs_macos do
```

Comment:
Don't want to add any more integration tests for this, I think, because they are super slow.